easybuild/easyconfigs/v/VESTA/VESTA-3.5.8-gtk3.eb

```diff
@@ -0,0 +1,28 @@
+# Based on https://code.it4i.cz/sccs/easyconfigs-it4i/-/blob/it4i-karolina/v/VESTA/VESTA-3.5.7.eb
+
+easyblock = 'Binary'
+
+name = 'VESTA'
+version = '3.5.8'
+versionsuffix = '-gtk3'
+
+homepage = "http://www.jp-minerals.org/vesta"
+description = """
+  VESTA is a 3D visualization program for structured models, 
+  volumetric data such as electron/nuclear densities, 
+  and crystal morphologies.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ["http://www.jp-minerals.org/vesta/archives/%(version)s"]
+sources = ["%(name)s-gtk3.tar.bz2"]
```

Comment:
@guacke Please let EasyBuild rename this file after download, so it also includes the version:

```python
sources = [{'download_filename': '%(name)s-gtk3.tar.bz2', 'filename': SOURCE_TAR_BZ2}]
```

(see also https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources)